src/api/hooks/useGetDelegationNodeInfo.ts

```diff
@@ -1,14 +1,23 @@
 import {AptosClient, Types} from "aptos";
 import {useState, useMemo} from "react";
 import {getValidatorCommission} from "..";
+import {DELEGATION_POOL_ADDRESS} from "../../constants";
 import {useGlobalState} from "../../GlobalState";
+import {useGetAccountResources} from "./useGetAccountResources";
 import {ValidatorData} from "./useGetValidators";
 import {useGetValidatorSet, Validator} from "./useGetValidatorSet";
 
+interface DelegationPool {
+  active_shares: {
+    total_coins: string;
+  };
+}
+
 type DelegationNodeInfoResponse = {
-  delegatedStakeAmount: string;
+  delegatedStakeAmount: string | undefined;
   networkPercentage?: string;
   commission: number | undefined;
+  isLoading: boolean;
```

Comment:
I don't think `isLoading` should be a type property. If you want to use it to indicate that the state is `isLoading` we should hold a local state (useState), similar to what we are doing here
https://github.com/aptos-labs/governance/blob/main/src/api/hooks/useSubmitTransaction.ts#L28